### PR TITLE
[Experiment] Use docker in Travis

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,6 +73,16 @@ matrix:
...
     
+    ### docker build test ###
+    - env: MODE=docker
+      services:
+        - docker
+      before_script:
+      before_install:
+      install:
+      script:
+        - make all-in-one
```
Comment (brimstone) [inline .travis.yml]: This need something that actually passes or fails. `all-in-one` just starts the environment with a satellite, a storagenode, and a gateway.

Comment (egonelbre) [reply .travis.yml]: I assumed it tests the setup?

Comment (brimstone) [reply .travis.yml]: `all-in-one` does not test anything in a way I define test. It does stand up a local environment with production grade images.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+RUN /app/satellite setup
+ENTRYPOINT ["/app/satellite", "run"]
+
+# Storage Node
+FROM alpine as storj-storagenode
+ENV SATELLITE_ADDR=
+EXPOSE 7776/udp 7777
+WORKDIR /app
+COPY --from=build-env /app/storagenode /app/storagenode
+RUN /app/storagenode setup
```
Comment (brimstone) [inline Dockerfile]: So, every storagenode has the same identity?

Comment (egonelbre) [reply Dockerfile]: Ah, right...

**Comment by egonelbre** [comment]
Closing because fixed in https://github.com/storj/storj/pull/900

**Pull request closed without merging**